docs/customising/make_a_new_theme.md

```diff
@@ -191,24 +203,19 @@ put the URL of your theme in your `THEME_URLS` config setting for your
 production server, just like you have done for this development one).
 
 There are different ways to commit your changes, but here's one method:
+<div class="attention-box helpful-hint">
+  Note that <em>inside</em> your Vagrant virtual machine, you won't be able to
+  access GitHub with your user settings, and so on. So you will need to do this
+  from "outside" the VM. This can trip you up because the command shell looks
+  very similar when you are inside a <code>vagrant&nbsp;ssh</code> session,
+  and when you are not.
+</div>
```

Comment:
Must be my version of Jekyll; just check my comments against the live site once deployed. :+1: